Roslyn.runsettings

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>  
+<RunSettings>  
+  <RunConfiguration>  
+    <TestAdaptersPaths>%userprofile%\.nuget\packages\xunit.runner.visualstudio\2.2.0-beta4-build1194</TestAdaptersPaths>  
```

Comment:
So, I went ahead and tried it, and this does seem to be unnecessary with #20333.  @jaredpar is that likely to get merged soonish?